frontend/Store.js

```diff
@@ -198,10 +200,14 @@ class Store extends EventEmitter {
   }
 
   changeSearch(text: string): void {
+    this.searchText = text;
+    this.emit('searchText');
+    this._emitSearchRootsChange(text);
+  }
+
+  filterSearchRoots(text: string): void {
     var needle = text.toLowerCase();
-    if (needle === this.searchText.toLowerCase()) {
```

Comment:
By the time it hits this line, `this.searchText` is guaranteed to be updated. Therefore `needle` will always be equal to `this.searchText`.